readme.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
-```js
+```
+$.get("http://www.kittengifs.com/gifs")
 
 ```
```

Comment:
We haven't used `$.get` in class, but it is a valid jQuery method. We usually use either `$.ajax` or `$.getJSON`.

That being said, we still need to do something with the response after the call is made. An answer to this question could look like the following...

```js
$.ajax({
  url: "http://kittengifs.com/gifs"
  method: "get"
})
 .then(function(results){
   console.log(results.length)
 });
```

Let me know if you have any questions on AJAX and how to make a proper AJAX call. This is a key concept that we will keep using moving forward.